generate-roles.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import "github.com/bblfsh/sdk/uast"
+import "fmt"
+
+func main() {
+	lastRole := uast.Whitespace + 1
+	fmt.Println(
+		`////////////////////////////////////////////////////
+// Automatically generated by "generate-roles.go" //
+////////////////////////////////////////////////////
+
+#include "roles.h"
+
+static const char *no_rule = "role.empty";
+static const char *unknown_rule = "role.unknown";
+static const char *id_to_roles[] = {`)
+
+	for i := 0; i < int(lastRole); i++ {
+		roleID := uast.Role(i)
+		name := "role." + roleID.String()
```

Comment:
I talked with maximo about it, I tested with `:` but it didn't work quite good. So ok, no dot. I will remove it, but I am against it hahaha ;)